src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.db;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Storage {
+    public static final List<FruitTransaction> db = new ArrayList<>();
```

Comment:
it is better to use map here (we can conveniently store fruitName-currentAmount pairs)

if you want to use List then create some class that would contain `fruitName` and `amount` fields to represent our storage

```
List<Fruit> storage = new ArrayList();

class Fruit {
    String name;
    int amount;
}
```